apps/babanews/pages/index.tsx

```diff
@@ -31,7 +33,7 @@ const Frontpage = styled(PageContainer)`
 `
 
 export default function Index() {
-  return <Frontpage slug={''} />
+  return <Frontpage slug={'home'} />
```

Comment:
We should use a page without a slug ideally.

Reply:
removed